docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -16,6 +16,18 @@ To connect to a ClickHouse server, you need:
 * Network access from the Trino coordinator and workers to the ClickHouse
   server. Port 8123 is the default port.
 
+.. note::
+
+    Trino uses the new ClickHouse driver(``com.clickhouse.jdbc.ClickHouseDriver``)
+    by default after version 369, but the new driver only supports ClickHouse server with version >= 20.7.
```

Comment:
We generally don't document the changed version in the connector's page. It will be mentioned in release notes.